website/api/helpers/strings/to-html.js

```diff
@@ -112,6 +112,17 @@ module.exports = {
       return `<blockquote><img src="/images/icon-info-16x16@2x.png" alt="An icon indicating that this section has important information"><div class="d-block">`+quote+`</div></blockquote>\n`;
     };
 
+    // Custom renderer function to enable checkboxes in Markdown lists.
+    customRenderer.listitem = function(text, task, checked){
+      if(!task){ // « If a list item does not have a checkbox, we'll render it normally.
+        return `<li>${text}</li>`;
+      } else if(checked) {// If this checkbox was checked in Markdown (- [x]), we'll add a disabled checked checkbox, and hide the original checkbox with CSS
+        return `<li purpose="checklist-item"><input disabled="" type="checkbox" checked><span purpose="task">${text}</span></li>`;
+      } else {// If the checkbox was not checked, we'll add a checkbox that can be checked by users, and hide the original checkbox with CSS.
+        return `<li purpose="checklist-item"><input type="checkbox"><span purpose="task">${text}</span></li>`;
```

Comment:
Is `text` true plain text, with html already escaped?  Please test that by hand because docs sometimes lie- use `&` characters. If truly is plaintext, all good.  But if it’s html, then name the variable accordingly.

I would guess it is html.

Reply:
Also, shouldn’t the checkbox be disabled in both cases?  Users will never be able to save their checkings and uncheckings

Reply:
@eashaw Rename `text` to `innerHtml`

Reply:
rename `task` to `hasCheckbox`,
consider renaming `checked` to `isChecked`